src/modules/IsnsServer.rb

```diff
@@ -351,7 +351,9 @@ def isnsadm(params, ret_result = false)
 
     def isnsadm_control
       if !@isctrlnode
-        @isctrlnode = isnsadm("--register control")
+        res = isnsadm("--register control", ret_result = true)
+        @isctrlnode = res["exit"] == 0
```

Comment:
What I am fixing here is:
I bring 'res' in that method, that was not there before. So when the error message was about to be returned, it died because an undefined value here:
          y2error("Registering as control node failed: #{res["stderr"]}; #{res["stdout"]}")